ansible/roles/miniflux/defaults/main.yml

```diff
@@ -132,14 +132,6 @@ miniflux__database_password: '{{ lookup("password", secret + "/postgresql/" +
 # installation definition can be found in the
 # :envvar:`miniflux__golang__dependent_packages` variable.
 
-# .. envvar:: miniflux__upstream_gpg_key [[[
-#
-# The fingerprint of the GPG key which is used to sign the Miniflux
-# releases. It will be used to verify the downloaded signature file as
-# well as the :command:`git` tags in the source repository.
-miniflux__upstream_gpg_key: '5A7A 89AC F055 24CA A0F3  6F9B AEB7 A164 0710 8DB5'
```

Comment:
Miniflux doesn't offer GPG key for their APT packages anymore. They are using `trusted=yes` attribute in the [APT configuration](https://miniflux.app/docs/debian.html).